test/lib/rubycritic/core/smell_test.rb

```diff
@@ -69,4 +69,27 @@
       smell1.must_equal smell2
     end
   end
+
+  describe '#dasherized_type' do
```

Comment:
I believe we should test this method implicitly through doc_url  - how about moving those tests there?